site/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bright-js-framework-docs",
+  "private": true,
+  "description": "Documentation for bright-js-framework",
+  "version": "1.0.0",
+  "author": "Chris Devereux",
+  "dependencies": {
+    "gatsby": "^1.9.45",
+    "gatsby-link": "^1.6.17",
+    "gatsby-plugin-react-helmet": "^1.0.5",
+    "lodash": "^4.17.4",
+    "material-ui": "^0.19.4",
+    "react-markdown": "^2.5.0"
+  },
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "MIT",
+  "main": "n/a",
+  "scripts": {
+    "build": "gatsby build --prefix-paths",
+    "develop": "gatsby develop",
+    "format": "prettier --trailing-comma es5 --no-semi --single-quote --write 'src/**/*.js'",
```

Comment:
I really like this approach to standardising code formatting. Is this a manual invocation?

Reply:
This is part of the docsite starter kit and only applies to that subdirectory. There's a separate PR open to add a githook that runs a code formatter on the framework codebase, though